pkg/gencmd/util/string.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strings"
+
+	"github.com/jinzhu/inflection"
+	"github.com/serenize/snaker"
+)
+
+type String struct {
+	Camel struct {
+		Plural   string
+		Singular string
+	}
+	CamelLower struct {
+		Plural   string
+		Singular string
+	}
+	Snake struct {
+		Plural   string
+		Singular string
+	}
+}
+
+func Inflect(in string) (out String) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function Inflect should have comment or be unexported